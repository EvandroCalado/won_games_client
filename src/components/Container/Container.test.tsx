import { render } from '@testing-library/react';
import { Container } from '.';

describe('Container', () => {
  it('should render a heading', () => {
    const { container } = render(<Container>container</Container>);

    expect(container.firstChild).toHaveClass('mx-auto max-w-7xl px-4');
  });

  it('should render a snapshot', () => {
    const { container } = render(<Container>container</Container>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
