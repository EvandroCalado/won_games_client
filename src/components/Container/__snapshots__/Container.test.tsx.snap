// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render a snapshot 1`] = `
<div
  class="mx-auto max-w-7xl px-4 py-8"
>
  container
</div>
`;
